backend/deepchecks_monitoring/api/v1/model.py

```diff
@@ -671,10 +675,49 @@ async def retrieve_connected_models(session: AsyncSession = AsyncSessionDep) ->
         .outerjoin(ingestion_info, ingestion_info.c.model_id == Model.id)
     )).all()
 
-    return [
-        ConnectedModelSchema.from_orm(record)
-        for record in records
-    ]
+    connected_models: t.List[ConnectedModelSchema] = []
+
+    for record in records:
+        model_id = record.id
+
+        model: Model = (
+            await session.execute(sa.select(Model).where(Model.id == model_id).options(selectinload(Model.versions)))
+        ).scalars().first()
```

Comment:
not important, but `session.scalar(...)` can be used instead of `(session.execute(...)).scalars().first()`